### DISCUSSION TITLE
Boundary attack implementation in ART 1.8

### DISCUSSION THREAD

**Discussion by Abhishek2271** [Trusted-AI/adversarial-robustness-toolbox#1372] [Q&A] [answered]
Hi,

As per this feature request: https://github.com/Trusted-AI/adversarial-robustness-toolbox/issues/1215

It seems that the ability to perform boundary attack without any models at hand was added to ART 1.8. But I could not find any resources on how to implement it.
Are there any help files that can be used to do so?

**Comment by beat-buesser** [comment, marked as answer]
Hi @Abhishek2271 This notebook demonstrates the new estimator functionality based on look-up tables: https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/notebooks/classifier_blackbox_lookup_table.ipynb

Contrary to the title of this discussion, this estimator is not specifically related to `BoundaryAttack` but implemented as feature in `BlackBoxClassifier`.

**Reply by Abhishek2271** [reply]
Hi @beat-buesser,
Thank you for the reply. The notebook is really helpful in understanding the implementation (although I am still looking into it to fully understand it).
Thank you again for the help.

**Reply by beat-buesser** [reply]
@Abhishek2271 That's great! Let us know anytime if you have questions or suggestions!
